addons/website/static/src/snippets/s_image_gallery/001.scss

```diff
@@ -220,27 +223,28 @@
         }
         &.s_image_gallery_indicators_rounded {
             .carousel {
-                ul.carousel-indicators li {
-                    border-radius: 50%;
+                .carousel-indicators {
+                    img, i {
+                        border-radius: 50%;
+                    }
                 }
             }
         }
         &.s_image_gallery_indicators_dots {
             .carousel {
-                ul.carousel-indicators {
+                .carousel-indicators {
                     height: 40px;
                     margin: auto;
 
-                    li {
+                    img, i {
                         max-width: 8px;
                         max-height: 8px;
                         margin: 0 6px;
                         border-radius: 10px;
-                        background-color: $black;
-                        background-image: none !important;
+                        filter: brightness(0);
 
                         &:not(.active) {
-                            opacity: .4;
+                            filter: brightness(0.3);
```

Comment:
Not sure this trick is acceptable. Before, replacing the background image with a background color black was great. Now as we have to do it with `<img/>` so I just play with the brightness. The problem is that if the user look very closely, he could see that the unactive dots are not completely gray... (because it's still the image)...

Reply:
Maybe try to use border + border-box.
It will completely hide the images and like that it will be easier for users to change the color of the dots in css.

```css
    border: 5px solid #000000;
    box-sizing: border-box;
```

Reply:
Another possibility would be to put a svg as the image source.